packages/cardano-services-client/test/HandleProvider/handleProvider.test.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
This file should be renamed to make clear it tests stuff from `KoraLabsHandleProvider.ts` - [ref](https://github.com/input-output-hk/cardano-js-sdk/pull/732#discussion_r1205665133)

Reply:
dde35b14797987 should address this